src/main/resources/static/api.yaml

```diff
@@ -1901,6 +1901,26 @@ components:
           $ref: '#/components/schemas/Gender'
         hasTransactionPin:
           type: boolean
+        verifiedIdentifiers:
```

Comment:
Should we not also have "unverifiedIdentifiers"? we have that for discovery.

Reply:
Added